### ISSUE TITLE
Video mode 14 - 2560x1440@60 not working.

### ISSUE THREAD

**Issue by patjohnny** [furrtek/Arcade-TMNT_MiSTer#10]
Tried video mode 14 and it only displays in 1080p.

**Comment by Toryalai1** [comment]
The framework needs to be updated.

**Comment by patjohnny** [comment]
You mean by Sorglig?

**Comment by MikeS11** [comment]
I can put in a PR to add the new framework including the YC module for composite and Svideo but won't be home for a couple days.

**Comment by Toryalai1** [comment]
No, the framework in this repo needs to be updated. It will be updated soon

**Comment by patjohnny** [comment]
Nice. I was also wondering about yc but you answered it. I'll close this issue. Thanks.
